### PR TITLE
Updated to discuss OpenCost

### PR DESCRIPTION
**Pull request by mattray**
Also updated go.mod link, probably need to include front-end deps too once the UI is being deployed

### CODE REVIEW

File: open-source-deps.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 Open-Source
 ===========
 
-Kubecost has an open core business model. 
+Kubecost has open source at the center of our business model.
```
Comment (jessegoodier) [inline open-source-deps.md:4]: open-core is a specific business model. I would at least call that out?

like:
```
Kubecost has open source at the center of our business model (open-core). 
```

But I agree that it was previously vague and assumed the reader understood open-core.

File: open-source-deps.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 Open-Source
 ===========
 
-Kubecost has an open core business model. 
+Kubecost has open source at the center of our business model. 
 
-Our commercial project is based on our cost-model open-source project, which has an Apache 2.0 license. 
-Here is a list of the direct [open-source dependencies](https://github.com/kubecost/cost-model/blob/master/go.mod#L9) that we also include as part of our project.
+Our commercial project is built on top of the [OpenCost CNCF Sandbox project](https://www.cncf.io/projects/opencost/), which has an Apache 2.0 license.
```
Comment (dwbrown2) [inline open-source-deps.md:6]: Commercial products or product instead of project?

File: open-source-deps.md (hunk 1 above)
Comment (srpomeroy) [reply open-source-deps.md]: Agreed, added back open-core.

File: open-source-deps.md (hunk 2 above)
Comment (srpomeroy) [reply open-source-deps.md]: Yep!

**Pull request merged**